components/tools/OmeroPy/src/omero/plugins/web.py

```diff
@@ -82,15 +84,18 @@ def _configure(self, parser):
 
         config = parser.add(
             sub, self.config,
-            "Output a config template for server"
-            " ('nginx' or 'apache' for the moment)")
-        config.add_argument("type", choices=("nginx", "apache"))
+            "Output a config template for web server\n"
+            "  nginx: Nginx system configuration for inclusion\n"
+            "  nginx-development: Standalone user-run Nginx server\n"
+            "  apache: Apache 2.2 with mod_fastcgi\n"
+            "  apache-fcgi: Apache 2.4+ with mod_proxy_fcgi\n")
+        config.add_argument("type", choices=(
+            "nginx", "nginx-development", "apache", "apache-fcgi"))
         config.add_argument(
             "--http", type=int,
             help="HTTP port for web server (not fastcgi)")
         config.add_argument(
-            "--system", action="store_true",
```

Comment:
Yes, but up until now the only way to get "system" functionality was to pass `--system` in which case we can assume there are a number of scripts out there that pass it, right? If so, and we **leave** `--system` making it the default, only users of (to be added) `--development` will be broken.